Author: Pino Toscano
Description: sys/soundcard.h includes linux/soundcard.h on linux,
 so there is no need for complications
--- puredata.orig/src/s_audio_oss.c
+++ puredata/src/s_audio_oss.c
@@ -5,11 +5,7 @@
 
 /* this file inputs and outputs audio using the OSS API available on linux. */
 
-#if defined(__FreeBSD_kernel__)
-# include <sys/soundcard.h>
-#else
-# include <linux/soundcard.h>
-#endif
+#include <sys/soundcard.h>
 
 #include "m_pd.h"
 #include "s_stuff.h"
